### PR TITLE
Flex v2.0 support paste

### PR DESCRIPTION
**Pull request by Lightskyz**
Replaced Material UI by Twilio Paste

### CODE REVIEW

File: src/ChatMessageCannedResponsePlugin.tsx [hunk 1]
```diff
@@ -24,10 +27,28 @@ export default class ChatMessageCannedResponsePlugin extends FlexPlugin {
...
     const crmPanelView = false;
     const queryClient = new QueryClient();
+
     const options: Flex.ContentFragmentProps = {
-      if: (props: any) => flex.TaskHelper.isChatBasedTask(props.task),
+      if: (props: any) => {
+        // In the TaskCanvas, we have access to the task directly. 
+        // In the AgentDesktopView, we don't, however we have access to all the tasks and the selected one that we could retrieve
+        // When completing the task, selectedTaskSid still exist but the task in the map has been removed, so we have to check the size of it
+        console.log("Component props: ", props);
```
Comment (mschmitt19) [inline src/ChatMessageCannedResponsePlugin.tsx]: We can remove the `console.log` statements

File: src/components/ChatDropdown/ChatDropdown.tsx [hunk 2]
```diff
@@ -1,114 +1,70 @@
...
+
   return (
-    <Grid container className={classes.root}>
+    <Box paddingBottom="space60" margin="auto">
       <TaskContext.Consumer>
         {(context) => (
           <>
             {status === "loading" ? (
-              <CircularProgress className={classes.progress} />
+              <Button variant="primary" loading>Loading</Button>
```
Comment (mschmitt19) [inline src/components/ChatDropdown/ChatDropdown.tsx]: Should we switch this `Button` component to a `SkeletonLoader` for the loading state?

**Review by mschmitt19 — DISMISSED** [review]
Left a couple comments, but overall great job! Looks great.

File: src/components/ChatDropdown/ChatDropdown.tsx (hunk 2 above)
Comment (johnfischelli) [reply src/components/ChatDropdown/ChatDropdown.tsx]: Done!

**Review by johnfischelli — APPROVED** [review]

**Pull request merged**